src/App.tsx

```diff
@@ -55,6 +57,35 @@ loader.init().then((monaco) => {
   })
 })
 
+const getAppDataFromSearchParams = (): AppData => {
+  const urlParams = new URLSearchParams(window.location.search)
+  const encodedAppData = urlParams.get('appData')
+  if (!encodedAppData)
+    return {
+      schema: '',
+      values: [],
+    }
+
+  const decompressedAppData =
```

Comment:
Do not mix `encoded` and `decompressed` prefixes.
Given the context either would be ok (the library does both in one step, we should use a different method if we want to highlight both but there's really no need to imho), but be consistent.

Reply:
done